From 5f91a5cd7d854e9e66ae455559a9cf0d6d78f60c Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Tue, 19 Dec 2023 14:03:51 +0100
Subject: [PATCH 1065/1141] dt-bindings: media: st,stm32-dcmipp: add
 'power-domains' property

STM32 DCMIPP may be in a power domain which is the case for the STM32MP25
based boards. Allow a single 'power-domains' entry for STM32 DCMIPP.

Fix dtbs_check warning:
 Unevaluated properties are not allowed ('power-domains' was unexpected)

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Ied0fe7bf5fc4f42828e2b0e8a9192efe045ce403
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/347868
Reviewed-by: Alain VOLMAT <alain.volmat@foss.st.com>
Reviewed-by: Hugues FRUCHET <hugues.fruchet@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Alain VOLMAT <alain.volmat@foss.st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
---
 Documentation/devicetree/bindings/media/st,stm32-dcmipp.yaml | 3 +++
 1 file changed, 3 insertions(+)

--- a/Documentation/devicetree/bindings/media/st,stm32-dcmipp.yaml
+++ b/Documentation/devicetree/bindings/media/st,stm32-dcmipp.yaml
@@ -35,6 +35,9 @@ properties:
   resets:
     maxItems: 1
 
+  power-domains:
+    maxItems: 1
+
   port:
     $ref: /schemas/graph.yaml#/$defs/port-base
     unevaluatedProperties: false
